### ISSUE TITLE
Ejercicio 1 del fichero `demo_random_nystroem_features_classification.ipynb`

### ISSUE THREAD

**Issue by IkerVillegas** [BlancaCC/MetodosFuncionalesAprendizajeAutomatico#15]
Afer having scaled the data to the range $[-1, 1]$ using sklearn's *MinMaxScaler*, determine the optimal hyperparameters (gamma, nu) and the optimal number of features for the different classifiers consideres in this example using CV on the training set. Compare the prediction error of the different classifiers in the test set $\left\{X_{test}, y_{test} \right\}$. 

References:
https://scikit-learn.org/stable/modules/preprocessing.html

https://scikit-learn.org/stable/auto_examples/model_selection/plot_grid_search_digits.html#sphx-glr-auto-examples-model-selection-plot-grid-search-digits-py
